lib/cli/src/Cardano/CLI.hs

```diff
@@ -1577,22 +1577,18 @@ data Verbosity
     deriving (Eq, Show)
 
 data LogOutput
-    = LogToStdout Severity
+    = LogToStdStreams Severity
+    -- ^ Log to console, with the given minimum 'Severity'.
+    --
+    -- Logs of Notice or higher severity will be output to stderr. Info or lower
```

Comment:
If `Notice` is ["Normal but significant conditions"](https://github.com/input-output-hk/cardano-wallet/wiki/Logging-Guidelines), I guess one _could_ argue it should belong to stdout.

Reply:
Ah, @rvl you said that too.

>  <= Notice on stdout 

Will change then.